### PR TITLE
Add placeholder value to titles

### PR DESCRIPTION
**Pull request by VeraZab**
closes: #217 
ex

### CODE REVIEW

File: src/components/widgets/text_editors/MultiFormatTextEditor.js [hunk 1]
```diff
@@ -209,7 +209,14 @@ class MultiFormatTextEditor extends Component {
       return null;
     }
 
-    const {onChange, placeholder, value, localize: _} = this.props;
+    const {onChange, localize: _} = this.props;
+    let {placeholder, value} = this.props;
+
+    if (value.startsWith('Click to enter') && value.endsWith('title')) {
```
Comment (VeraZab) [inline src/components/widgets/text_editors/MultiFormatTextEditor.js]: or @alexcjohnson would you know of a better way for me to get that default string when title's not set?

**Comment by VeraZab** [comment]
ready for review @alexcjohnson @bpostlethwaite

File: src/components/widgets/text_editors/MultiFormatTextEditor.js (hunk 1 above)
Comment (alexcjohnson) [reply src/components/widgets/text_editors/MultiFormatTextEditor.js]: Mmm, that way isn't going to work with i18n anyway. Is it possible to compare the value from `fullLayout` with the value from `layout`? If they're different, I think that's a robust signal that you have the placeholder. And if you explicitly typed in the placeholder, then it's not really a placeholder anymore :)

Comment (alexcjohnson) [reply src/components/widgets/text_editors/MultiFormatTextEditor.js]: If we're not comfortable using the plotly.js placeholder verbatim, I think we'd have to explicitly make and localize a new placeholder for each attribute that uses this. Doesn't seem worth it to me, frankly; I'd either use the one from plotly.js or use a standard placeholder for every text entry box.

Comment (VeraZab) [reply src/components/widgets/text_editors/MultiFormatTextEditor.js]: yeah, we could use those plotly.js strings verbatim, it's easier for localization too, like you mention, so I'll put those strings as placeholders then, not values

**Comment by VeraZab** [comment]
ex

container is giving me that attribute's immediate container, so this seems to work ok to me
@alexcjohnson ?

**Comment by VeraZab** [comment]
oups, I forgot to also check against attr
so `attr === 'title' && !container.title`, seems to be what I need to check against

**Comment by alexcjohnson** [comment]
Great! I'm happy with this, thanks for iterating through it with me. 💃

**Comment by VeraZab** [comment]
thanks!

**Pull request merged**